Functions.Templates/Templates/DurableFunctionsHttpStart-CSharp-1.x/metadata.json

```diff
@@ -14,7 +14,7 @@
   ],
   "extensions": [
     {
-      "id": "Microsoft.Azure.WebJobs.Extensions.DurableTask", "version": "1.8.7"
+      "id": "Microsoft.Azure.WebJobs.Extensions.DurableTask", "version": "2.5.1"
```

Comment:
Need to keep this as is

Reply:
updated the code. Thanks for catching it.